### PR TITLE
Skip welcome e-mail

### PR DESCRIPTION
**Pull request by rocktavious**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -90,4 +90,4 @@ require (
 )
 
 // Uncomment for local development
-// replace github.com/opslevel/opslevel-go/v2023 => ./submodules/opslevel-go/
+replace github.com/opslevel/opslevel-go/v2023 => ./submodules/opslevel-go/
```
Comment (DougEdey) [inline go.mod:93]: Is this because we're not doing a release in advance or should this change!

**Review by DougEdey — APPROVED** [review]
Should anything from the dependency update go in the change log?

File: go.mod (hunk 1 above)
Comment (rocktavious) [reply go.mod:93]: its so the tests will pass.  Upon release we undo this and updated opslevel-go to the released version that happened that day

**Comment by rocktavious** [comment]
Thats an implict update coming from opslevel-go

**Pull request merged**